README.md

```diff
@@ -1 +1,64 @@
-# react-redux
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Africa Countries
+This is a web application that shows all African countries with their demographic data such as total population, capital cities, and currency they use.
+
+
+## Getting Started
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## Live link
+[Live Demo Link](https://africa-metadata.herokuapp.com/)
+
+## Live link
+[Video Link](https://www.loom.com/share/d1348d4f96ba4a498083422926882105)
```

Comment:
- [ ] Just like the previous reviewer said, the video presentation is too short and doesn't contain other presentation requirements. Kindly record another video and highlight features of the project (1) A piece of code you are proud of. (2) The video should contains a clear and objective description of the project in professional English. The topic and purpose of the project should be made clear. (3) The video should contains a demo of the project, highlighting all the project features.